reconcile/utils/terrascript_client.py

```diff
@@ -3892,12 +3917,72 @@ def populate_tf_resource_asg(self, resource, namespace_info):
                 {
                     "resource_type": "volume",
                     "tags": tags
-                }
-            ]
+                }]
         }
 
-        region = common_values.get('region') or \
-            self.default_regions.get(account)
+        image = common_values.get('image')
```

Comment:
OMG, this block is HUGE. Can you please split it into smaller chunks we can read in isolation?